fake_switches/brocade/command_processor/enabled.py

```diff
@@ -67,7 +67,7 @@ def do_skip_page_display(self, *args):
         pass
 
     def do_write(self, *args):
-        pass
+        self.switch_configuration.commit()
```

Comment:
What if commit did something (dump config to disk, or something), would you still put the sleep in the different implementations or in switch_configuration ?